www/Dockerfile

```diff
@@ -29,6 +40,24 @@ FROM nginx:mainline-alpine
 # Add dependencies for our scripts
 RUN apk add --no-cache bash openssl inotify-tools
 
+# mountable content for web app (remove default stuff in there)
+RUN rm /usr/share/nginx/html/*
+COPY html/503.html /usr/share/nginx/html
+COPY --from=webapp /usr/src/app/dist /usr/share/nginx/html
+
+# configure openssl to use OQS provider
+COPY --from=oqs-provider /oqs-provider/_build/lib/oqsprovider.so /
+RUN cat <<EOF  >> /etc/ssl/openssl.cnf
```

Comment:
This is an append operation. Do we know what's in the file before this?

Reply:
openssl's config as shipped by Debian.